config/routes.rb

```diff
@@ -7,9 +7,46 @@
 
   # IRI Routing #
   ###############
-  #
   # as per Loc/Id definition
 
+  # Special (/ref-based) Loc/Id routes
+  specified_get '/ref/:reference/:repository_id/*locid' => 'ontologies#show',
+    as: :ontology_iri_versioned,
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.